actions/banked-months-updater.vbs

```diff
@@ -308,15 +313,18 @@ Show_dialog = False
 
 For i = 0 to Ubound(banked_months_array, 2)
 	'Determines if updates aren't going to be made to STAT/WREG
-	If banked_months_array(used_all_abawd_mo_const, i) = True then
-		update_wreg = True
-	Elseif banked_months_array(member_in_error_const, i) <> "" then 
-		update_wreg = True 	
+	'If banked_months_array(used_all_abawd_mo_const, i) = True then
+	'	update_wreg = True
+	'Else
+    If banked_months_array(member_in_error_const, i) = "" then 
+        update_wreg = True 	
+        spec_memo = True 
+        case_note = True 
 	End if 
 
-	If banked_months_array(used_all_banked_mo_const, i) = False then spec_memo = True 
-	If update_wreg = False then spec_memo = False 
-	If update_wreg = True then case_note = True 
+	'If banked_months_array(used_all_banked_mo_const, i) = False then spec_memo = True 
+	'If update_wreg = False then spec_memo = False 
+	'If update_wreg = True then case_note = True 
```

Comment:
Line 356 Could use a little extra room if we anticipate real cases having this many months listed as it is cutting off the last month
![banked months](https://github.com/Hennepin-County/MAXIS-scripts/assets/129224183/5bf3646e-3337-49ec-8ba1-d0b6214893ff)

Reply:
![banked months](https://github.com/Hennepin-County/MAXIS-scripts/assets/129224183/d6bffaaa-cdc2-4263-9316-aa82d72fe478)
Also wondering for line 358 if banked_months_array(banked_mo_string_const, i)) should default to 0 or something when the string is blank